src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write code here
+const list = document.querySelectorAll('li');
+
+function sortList(elements) {
+  const elementArray = Array.from(elements);
+
+  elementArray.sort((a, b) => {
+    const x = salaryToNumber(b.getAttribute('data-salary'));
+    const y = salaryToNumber(a.getAttribute('data-salary'));
+
+    return x - y;
+  });
+
+  const ul = document.querySelector('ul');
+
+  elementArray.forEach(x => ul.appendChild(x));
+}
+
+sortList(list);
+
+function getEmployees(elements) {
+  const employeesList = [];
+
+  for (let i = 0; i < elements.length; i++) {
+    employeesList.push({
+      name: elements[i].innerText,
+      position: elements[i].getAttribute('data-position'),
+      salary: elements[i].getAttribute('data-salary'),
+      age: elements[i].getAttribute('data-age'),
+    });
+  }
+
+  return employeesList;
+}
+
+function salaryToNumber(string) {
+  return string.slice(1).replaceAll(',', '');
+}
+
+getEmployees(list);
```

Comment:
This is OPTIONAL but we could [curry](https://javascript.info/currying-partials)  the results
1. first we need to make sortList return data that it's working on:
<img width="444" alt="image" src="https://github.com/mate-academy/js_task_parse_list_DOM/assets/87986955/acbc7759-f0b9-40b5-97fc-1e6f017ec4b5">

Reply:
2, now we can pass invocation of the sortList to getEmployees
<img width="287" alt="image" src="https://github.com/mate-academy/js_task_parse_list_DOM/assets/87986955/92761381-36b5-46b9-a85b-c3852d1eb7c4">

Reply:
Or do something even crazier, like implementing curry helper
<img width="391" alt="image" src="https://github.com/mate-academy/js_task_parse_list_DOM/assets/87986955/58bdbff6-92f4-487b-a737-3f59329ee937">

Reply:
then use it and pipe returned values from one function to another:
<img width="453" alt="image" src="https://github.com/mate-academy/js_task_parse_list_DOM/assets/87986955/652076ff-c50d-4f83-92ea-0e3ae19aa2bf">